smart_campus/smart_campus/settings.py

```diff
@@ -19,7 +19,7 @@ def get_env_variable(var_name):
     try:
         return os.environ[var_name]
     except KeyError:
-        error_msg = f'Environment variable "{var_name}" should be set!'
+        error_msg = 'Environment variable "{var_name}" should be set!'
```

Comment:
The `f` is NOT a typo.
It is the new string formatting method since Python 3.6 (Please check [PEP 498](https://www.python.org/dev/peps/pep-0498/)).
You can choose not use this method for backward compatibility.
But, you should not remove it which disables string formatting.